### PR TITLE
Link the SUL logo to the library website

### LINKED ISSUE #134: Link SUL logo to library website
Broken out from #66 
Change the link for part "A" -- currently it points to the app's homepage

A: Replace the "ArcLight" logo text with the SUL logo image - same version used in Exhibits. This should be linked to go to https://library.stanford.edu/ (which is different than Exhibits).

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #134

### CODE REVIEW

File: spec/requests/home_page_spec.rb [hunk 1]
```diff
@@ -10,6 +10,11 @@
   it "is a custom page that introduces the project" do
     get "/"
     expect(response).to have_http_status(:ok)
+
+    # Header
+    expect(page).to have_link 'Virtual Tribunals', href: 'https://library.stanford.edu/'
```
Comment (marlo-longley) [inline spec/requests/home_page_spec.rb:15]: @jcoyne Just noting that the link text says Virtual Tribunals when the link routes to Stanford Libraries/
The markup ends up being `<a class="navbar-brand mb-0 navbar-logo" href="https://library.stanford.edu/">Virtual Tribunals</a>` -- it works so it's fine for now but could be confusing to some / later.

**Comment by jcoyne** [comment]
Yeah, that's because Arclight uses `application_title` for the name of that link: https://github.com/projectblacklight/arclight/blob/main/app/views/shared/_header_navbar.html.erb#L4

**Comment by marlo-longley** [comment]
@jcoyne OK that makes sense -- I can approve this and open another ticket, or we can consider this issue / PR blocked until then.

**Comment by marlo-longley** [comment]
@jcoyne This looks good -- the SUL external link works and all the titles are correct on the different screens!
Thank you for sheparding through all the upstream changes that were needed to make this seemingly small thing happen.

There is a conflict in one spec file -- can you rebase?

**Comment by jcoyne** [comment]
@marlo-longley rebased.

**Review by marlo-longley — APPROVED** [review]

**Pull request merged**